### ISSUE TITLE
Silhouette single pass instanced

### ISSUE THREAD

**Issue by JurjenBiewenga** [ValveSoftware/steamvr_unity_plugin#169]
Currently the Silhouette shader does not support single pass instanced rendering, are there plans for the shader to support this?

**Comment by zite** [comment]
Yeah, I'd like to get this fixed. I'll be honest though, it's not a super high priority right now.
